content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+In the best case the user will gain new insights and deeper understanding of how each learner works, where it’s advantages are and also very importantly the cases when the learner fails miserably.
+There are a lot of different settings we want to offer in the user interface, and so – to not remove the fun of our playground – a huge effort went into making the UI as aesthetically pleasing and as smooth as possible.
+To achieve this, a website template was downloaded from <a href="https://templated.co/">Templated</a> and used as a baseline for design. After extending the template with missing CSS classes, most of the used shiny widgets have been overwritten – or even completely replaced –, offering refreshingly new visuals for the well known shiny framework.
+For the smoothness part, an object oriented <a href="https://cran.r-project.org/web/packages/R6/index.html">R6</a> class system with reactive attributes was engineered for the backend to give a well-defined framework of which elements should trigger what evaluation; an otherwise extremely tiresome and error-prone task for dozens of different UI elements.
+After all ‘mlrPlayground’ may not be as fun as a real playground, but you are also not as likely to hurt yourself and it is definitely more entertaining than looking at boring pictures of learners in a book.
+<br/>
+
+## The features
+
+In the following an extended overview of almost all available features is presented with the help of animated pictures.
+
+###### Task selection
+When running the app and scrolling down you can basically see two big boxes with content: the left one is for the task and learner settings, the right one for an overview of the learner’s bevahiour.
+For the first look it may be of help (even though not necessary) to explore the tasks.
+After finding the right button (I leave this task to the motivated reader) a panel with all available datasets is extended over our screen.
+Here, we can choose the tasktype of either classification with two covariables or regression with one covariable and several other parameters influencing the dataset: size, noise and the train/test split.
+
+<img src="https://i.imgur.com/ZV1wlBE.gif" width=800 />
+<br/>
+
+###### Learner selection
+Back at the main panel we can then select a learner for our task – due to limited manpower not all available in mlr have made it into this selection, expect extensions in the future. Now, it may take a few seconds because several things are happening in the background:
+
+<ul>
+<li><p>a model is trained on the training set</p></li>
+<li><p>different kinds of performance measures are calculated on the test set</p></li>
+<li><p>a grid of points is used for inference to create a plot showing the prediction surface.</p></li>
```

Comment:
I assume this sentence might confuse the reader. Can it be rephrased in more easy words?

Also is there a reason why you are using HTML lists here?